src/main/java/com/julienvey/trello/impl/http/RestTemplateHttpClient.java

```diff
@@ -53,4 +53,14 @@ public URI postForLocation(String url, Object object, String... params) {
             throw new TrelloHttpException(e);
         }
     }
+
+    @Override
+    public <T> T delete(String url, Class<T> responseType, String... params) {
+        try {
+            return restTemplate.exchange(url, HttpMethod.DELETE, null, responseType, (Object[]) params)
```

Comment:
what is this null param for?  let's at least document this if there is definitely nothing to provide

Reply:
I would extract this  to a variable called EMPTY_BODY and assign null value to it for readability. this makes it very obvious that the param is BODY and we explicitly provide null there